rotkehlchen/chain/ethereum/utils.py

```diff
@@ -158,10 +162,16 @@ def should_update_protocol_cache(cache_key: CacheType, *args: str) -> bool:
     the process of querying it again.
     """
     with GlobalDBHandler().conn.read_ctx() as cursor:
-        last_update_ts = globaldb_get_cache_last_queried_ts_by_key(
-            cursor=cursor,
-            key_parts=[cache_key, *args],
-        )
+        if cache_key in UNIQUE_CACHE_KEYS:
+            last_update_ts = globaldb_get_unique_cache_last_queried_ts_by_key(
+                cursor=cursor,
+                key_parts=(cache_key, *args),  # type: ignore
```

Comment:
why the type ignore here? Add small comment explaining it's due to cache_key needing type specification for the function here